pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -130,6 +130,7 @@ type JaegerStatus struct {
 
 // Jaeger is the Schema for the jaegers API
 // +k8s:openapi-gen=true
+// +operator-sdk:gen-csv:customresourcedefinitions.displayName="Jaeger"
```

Comment:
Isn't this what ends up being added here?

https://github.com/jaegertracing/jaeger-operator/blob/8575d518e687b83f86298a8ca8234178be2a2409/deploy/olm-catalog/jaeger-operator/1.18.1/jaeger-operator.v1.18.1.clusterserviceversion.yaml#L119 

If so, then the CSV in this PR doesn't seem to match this...

Reply:
https://github.com/operator-framework/operator-sdk/issues/2575